src/Microsoft.Health.Fhir.R5.Core/Microsoft.Health.Fhir.R5.Core.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.8" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.8" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.8" />
-    <PackageReference Include="Hl7.Fhir.R5" Version="1.10.0-alpha-20200811-1" />
+    <PackageReference Include="Hl7.Fhir.R5" Version="1.9.0-beta-june2020" />
```

Comment:
Is this downgrade intended?

Reply:
Yes... this is a bit of a hack to get my PR to build. I was hoping to merge my dev branch to my feature branch before merging in master (which has [this change](https://github.com/microsoft/fhir-server/commit/270dcdc5c98537797c00c9e4751e690330f631c0)). Let me know if this sounds alright - I am open to taking a different approach!

Reply:
Got it, that seems fine.